packages/perseus/src/components/input-with-examples.tsx

```diff
@@ -19,7 +17,6 @@ import type {StyleType} from "@khanacademy/wonder-blocks-core";
 const {captureScratchpadTouchStart} = Util;
 
 type Props = {
-    type: "math" | "text" | "tex";
```

Comment:
The removal of this prop will cause type errors in webapp when Perseus is updated. We'll need to bump the major version for this.

Alternatively, we could just mark this prop as deprecated, e.g.

```ts
/**
 * @deprecated type has no effect and will be removed in a future release of Perseus
 */
```

 and release this change under a minor version bump. Then we could remove the prop in a future major release. I'd rather just bump the major version, though.

Reply:
I debated major vs minor (and even considered patch); I decided on minor since it's a change to a _component_ and not a _widget_. The component is not exported, so I don't believe the external API of Perseus is changing - we're just changing how widgets consume an internal component.

I already started the upgrade process in [Webapp](https://github.com/Khan/webapp/pull/22886) to see if it breaks anything; if it throws errors I'll bump to major.

Reply:
Ah cool! Yes, you are totally right about it not being exported. I would actually lean toward a patch release then, since I don't think this changes any observable behavior? I.e. we're not adding a feature, just deleting dead code.